eviatar/enableFeatures_createUsers.ps1

```diff
@@ -0,0 +1,54 @@
+﻿function Install-WindowsFeatures {
+    param (
+        [Parameter(Mandatory = $true)]
+        [string[]]$Features
+    )
+    
+    foreach ($Feature in $Features) {
+        Write-Host "Installing feature: $Feature"
+        Install-WindowsFeature -Name $Feature -IncludeAllSubFeature -IncludeManagementTools -Verbose
+    }
+}
+
+function Create-LocalUsers {
+    param (
+        [Parameter(Mandatory = $true)]
+        [PSObject[]]$Users
+    )
+    
+    foreach ($User in $Users) {
+        Write-Host "Creating user: $($User.username)"
+        $password = ConvertTo-SecureString -String $User.password -AsPlainText -Force
+        New-LocalUser -Name $User.username -Password $password -FullName $User.username -Description "Created by script"
+        
+        Write-Host "Adding user '$($User.username)' to Administrators group"
+        Add-LocalGroupMember -Group "Administrators" -Member $User.username

```

Comment:
What is I want to add this user to a different group and restrict his permissions?

Reply:
You can extend the function to accept an additional parameter for the group name, and include logic for restricting permissions.